styles/whatsapp-web/catppuccin.user.less

```
@@ -67,613 +81,144 @@
+
+    #WDS(persistent-activity-indicator, fade(@green, 90%));
+    #WDS(persistent-always-black, #000);
+    #WDS(persistent-always-white, #FFF);
+    #WDS(persistent-always-branded, @accent);
+
+    #WDS(systems-bubble-surface-incoming, if(@lighterMessages = 1, @surface1, @base));
+
+    #WDS(systems-bubble-surface-outgoing, mix(@accent, @base, @deemphasize));
+    #WDS(systems-bubble-content-deemphasized, fade(@text, 50%));
+
+    #WDS(systems-bubble-surface-overlay, @mantle);
+    #WDS(systems-bubble-surface-system, @mantle);
+
+    #WDS(systems-bubble-surface-e2e, mix(@yellow, @mantle, 5%));
+    #WDS(systems-bubble-content-e2e, @yellow);
+
+    #WDS(systems-bubble-surface-business, mix(@teal, @mantle, 5%));
+
+    #WDS(systems-chat-surface-composer, @base);
+    #WDS(systems-chat-background-wallpaper, @crust);
+    #WDS(systems-chat-foreground-wallpaper, @mantle);
+    #WDS(systems-chat-surface-tray, @base);
+    #WDS(systems-status-seen, fade(@text, 50%));
+
+    #WDS(components-platform-gesture-bar, fade(@crust, 50%));
+    #WDS(components-platform-status-bar, fade(@crust, 80%));
+    #WDS(components-surface-nav-bar, @base);
+
+    #WDS(app-wash, @surface0);
+    
+    #RGB(attachment-type-stickers-color, @green);
+    #RGB(attachment-type-polls-color, @yellow);
+    #RGB(attachment-type-contacts-color, @sky);
+    #RGB(attachment-type-camera-color, @pink);
+    #RGB(attachment-type-photos-color, @blue);
+    #RGB(attachment-type-documents-color, @mauve);
+    #RGB(attachment-type-audio-color, @peach);
+    /* \/ Note: Seems to be overwritten by a generated
+    CSS variable, so has no effect :( */
+    #RGB(attachment-type-event-color, @pink);
+    
+    #RGB(toast-background, @surface1);
+    #RGB(toast-text, @text);
+
+    #RGB(picker-background, @surface1);
+
+    #RGB(butterbar-blue-nux-background, @sky);
+    #RGB(butterbar-blue-nux-icon-background, darken(@sky, 30%));
+    #RGB(blue-light, @blue);
+
+    #RGB(gray-500, @overlay2);
+
+    #RGB(focus-animation-deeper, mix(@accent, @base, 30%));
+    
+    /* "About AI messages" icon */
+    .st0 {
       fill: @accent !important;
+      stroke: @surface0;
+    }
+    
+    /* TODO: Meta AI/other bot colours */
+    
+    input[type="time"]::-webkit-datetime-edit-hour-field:focus,
+    input[type="time"]::-webkit-datetime-edit-minute-field:focus,
+    input[type="time"]::-webkit-datetime-edit-second-field:focus,
+    input[type="time"]::-webkit-datetime-edit-ampm-field:focus,
+    input[type="date"]::-webkit-datetime-edit-year-field:focus,
+    input[type="date"]::-webkit-datetime-edit-month-field:focus,
+    input[type="date"]::-webkit-datetime-edit-day-field:focus {
+      background-color: @accent;
+      color: @accentText;
     }
   }
 }
 
+#WDS(@name, @color) {
+    --WDS-@{name}: @color;
+    --WDS-@{name}-RGB: #rgbify(@color)[];
+}
+
+#RGB(@name, @color) {
+    --@{name}: @color;
+    --@{name}-rgb: #rgbify(@color)[];
+}
```

```suggestion
#WDS(@id, @color) {
    --WDS-@{id}: @color;
    --WDS-@{id}-RGB: #rgbify(@color)[];
}

#RGB(@id, @color) {
    --@{id}: @color;
    --@{id}-rgb: #rgbify(@color)[];
}
```

This should fix the metadata error. I think it's an issue with the UserCSS metadata parser we use or something where because we use `@name` down here it gets confused.